### PR TITLE
Minor custom VNET doc updates re subnets

### PR DESCRIPTION
**Pull request by rocketraman**
Fixes issue #437

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/441)

### CODE REVIEW

**Comment by rocketraman** [comment]
cc: @colemickens

File: examples/vnet/README.md [hunk 1]
```diff
@@ -4,6 +4,14 @@
 
 These examples show you how to build a customized Docker enabled cluster on Microsoft Azure where you can provide your own VNET.
 
+By providing your own VNET, you can change the VNET address space from the default of 10.0.0.0/8 to a smaller subnet, like
+10.1.0.0/16. This may be useful if you have existing VNETs that you would like to peer with, or plan to have them in the future.
+Kubernetes will still use 10.0.0.0/8 internally within the cluster, but all the node ports will be within the modified VNET
```
Comment (anhowe) [inline examples/vnet/README.md]: I would like to see acknowledgement of the 3 kubernetes networks .  Here's a proposal for the last sentence?

For Kubernetes, a change of the address space only affects the IP addresses assigned to the masters and the agents.  The service network of 10.0.0.0/16 is still implemented by the kube-proxy in each host, and the pod network stays as 10.244.0.0/16.  This means that access to services from another VNET will have to be done via node port through the host addresses.

**Comment by rocketraman** [comment]
@anhowe Made changes along the lines you suggested.

**Comment by ashb** [comment]
Possibly not needed it #473 is merged instead.

**Comment by shrutir25** [comment]
@anhowe - are we good to merge this PR ?

**Comment by shrutir25** [comment]
@acs-bot test this please

**Comment by jackfrancis** [comment]
@rocketraman Please forgive the staleness of the responsiveness in this PR. @lachie83 Could you give the documentation copy in this PR a look over and see if its additional language makes sense for the current state of VNET support?

Thanks!

File: examples/vnet/README.md [hunk 2]
```diff
@@ -4,6 +4,16 @@
 
 These examples show you how to build a customized Docker enabled cluster on Microsoft Azure where you can provide your own VNET.
 
+By providing your own VNET, you can change the VNET address space from the default of 10.0.0.0/8 to a smaller subnet, like
+10.1.0.0/16. This may be useful if you have existing VNETs that you would like to peer with, or plan to have them in the future.
+For Kubernetes, a change of the address space only affects the IP addresses assigned to the master and agent nodes. The service
+network of 10.0.0.0/16 is still implemented by the kube-proxy in each host, and the pod network stays as 10.244.0.0/16. This
+means that access to services from another VNET will have to be done via node port through the host addresses, or via an
+internal load balancer (note that currently, internal load balancers need to be created manually).
```
Comment (SaaldjorMike) [inline examples/vnet/README.md]: When using Kubernetes, you can let Kubernetes create this internal load balancer. See https://github.com/kubernetes/kubernetes.github.io/pull/4082/files

Comment (rocketraman) [inline examples/vnet/README.md]: Yes, that capability has been added in upstream since this pull request was created. Updated.

**Comment by jackfrancis** [comment]
Closing in favor of a future, larger effort to provide comprehensive custom VNET on Kubernetes docs.

**Pull request closed without merging**